client/src/commands/restartFabricRuntime.ts

```diff
@@ -28,12 +29,16 @@ export async function restartFabricRuntime(): Promise<void> {
         title: 'IBM Blockchain Platform Extension',
         cancellable: false
     }, async (progress: vscode.Progress<{ message: string }>) => {
-        progress.report({ message: `Restarting Fabric runtime ${runtime.getName()}` });
+        let fabricRuntimeName: string = runtime.getName();
```

Comment:
you don't need this if you can just use `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME`